### PR TITLE
Invalid NEO4J_PLUGINS values now produces a helpful error message

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-793.surge.sh

File: modules/ROOT/pages/docker/operations.adoc [hunk 1]
```diff
@@ -337,3 +313,53 @@ For example, to install the APOC plugin (`apoc`) and the Neo Semantics plugin (`
...
+  -e NEO4J_AUTH=none \
+  --env NEO4J_PLUGINS='["apoc", "n10s"]'
+  neo4j:{neo4j-version-exact}
+----
+====
+
+[NOTE]
+====
+The `--user="$(id -u):$(id -g)"` argument is required to ensure that the plugins are installed with the correct permissions.
+====
```
Comment (jennyowen) [inline modules/ROOT/pages/docker/operations.adoc:365]: this is not true, you can delete the `--user="$(id -u):$(id -g)"` from all the examples and they should still work.

**Review by jennyowen — CHANGES_REQUESTED** [review]
This is mostly fine, there's just an incorrect statement about needing to use the `--user` flag to be deleted and some minor code cleanup.

File: modules/ROOT/pages/docker/operations.adoc (hunk 1 above)
Comment (jennyowen) [reply modules/ROOT/pages/docker/operations.adoc]: yes I think it's a good idea to remove the line from the examples.

**Review by jennyowen — APPROVED** [review]
wonderful!

**Pull request merged**